docker-static/base-app/entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# rsyslog needs to run so that lfmerge can log to /var/log/syslog
+/etc/init.d/rsyslog start
+
+# Now chain to Docker entrypoint from base php:apache image
+exec docker-php-entrypoint "$@"
```

Comment:
this is really slick.  I like how you have chained the two entry points together.  Well done.  I would have probably chained the CMD together but the entry point chaining is better.

Reply:
I checed and some of our Dockerfiles use CMD, so entrypoint chaining lets those work without change.